conf/config.js

```diff
@@ -2,7 +2,7 @@ const { auth } = require('google-auth-library')
 const { Token } = require('graphql')
 const { environment } = require('./settings')
 
-exports.mongoConnection = () => {
+/*exports.mongoConnection = () => {
```

Comment:
Removal of Commented Code: I've noticed that a lot of code is currently commented out, particularly around the MongoDB connection configuration and contracts. If this code is no longer necessary, I would recommend removing it to improve readability. If parts of this code are still useful for future reference, I suggest moving it into a separate reference document or saving it in commit history.

Explanation for Commented Code: If some parts of the commented code are still relevant, it would be helpful to add comments explaining why they are commented out and under what circumstances they might be uncommented.